api/nodes/serializers.py

```diff
@@ -1,42 +1,59 @@
 from rest_framework import serializers as ser
 
-from api.base.serializers import JSONAPISerializer, LinksField, Link
+from api.base.serializers import JSONAPISerializer, LinksField, Link, WaterbutlerLink
 from website.models import Node
 from framework.auth.core import Auth
 
 
 class NodeSerializer(JSONAPISerializer):
 
-    filterable_fields = frozenset(['title', 'description'])
+    category_choices = Node.CATEGORY_MAP.keys()
+    category_choices_string = ', '.join(["'{}'".format(choice) for choice in category_choices])
+    filterable_fields = frozenset(['title', 'description', 'public'])
 
     id = ser.CharField(read_only=True, source='_id')
     title = ser.CharField(required=True)
-    description = ser.CharField(required=False, allow_blank=True)
-    category = ser.ChoiceField(choices=Node.CATEGORY_MAP.keys())
+    description = ser.CharField(required=False, allow_blank=True, allow_null=True)
+    category = ser.ChoiceField(choices=category_choices, help_text="Choices: " + category_choices_string)
```

Comment:
Didn't realize ChoiceField was coming straight from DRF. 

What I'm saying is that it could/should be an implicit behavior of a ChoiceField that a ChoiceField instance's help_text attribute would be ```"Choices: " + ', '.join(["'{}'".format(choice) for choice in self.choices])```, rather than requiring external classes to generate human-readable help_text for them. 

NodeSerializer should never need to know about ```category_choices_string```. 

But this is just my opinion.